Add deprecation warnings for multi-node (#2855)
timescale/docs: _partials/_multi-node-deprecation.md

<Highlight type="warning">

> Multi-node support has been deprecated. TimescaleDB 2.13 is the last version that includes multi-node support. Multi-node support in 2.13 is available for PostgreSQL 13, 14 and 15.

Reviewer: Still a lot of "multi-node" but I think this could be easier to understand:

```suggestion
Multi-node support has been deprecated with TimescaleDB 2.13. This is the last version that includes multi-node support and it supports multi-node installations with PostgreSQL versions 13, 14, and 15.
```

Learn more about it [here][multi-node-deprecation].
</Highlight>

> [multi-node-deprecation]: https://github.com/timescale/timescaledb/blob/main/docs/MultiNodeDeprecation.md

Author: Not sure if adding a link this way will work as this is a partial - but I hope that it will work
